lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/ThirdPartyAPIServiceFactory.java

```diff
@@ -77,6 +77,8 @@ private ThirdPartyAPIService createDvaThirdPartyDocumentGateway(
         DvaCryptographyService dvaCryptographyService =
                 new DvaCryptographyService(configurationService);
 
+        RequestHashValidator requestHashValidator = serviceFactory.getRequestHashValidator();
```

Comment:
Does this need to be in the service factory and the API service factory. Can we not just create it here?